codegen/render/kube_types_renderer.go

```diff
@@ -73,7 +68,7 @@ var ControllerTemplates = inputTemplates{
 func RenderApiTypes(grp Group) ([]OutFile, error) {
 	defaultKubeCodeRenderer := KubeCodeRenderer{
 		templateRenderer:    DefaultTemplateRenderer,
-		TypesTemplates:      TypesTemplates(grp.Generators.HasDeepcopy()),
```

Comment:
So after this change is `HasDeepcopy()` not respected anymore?

Reply:
no projects currently leverage the functionality, i think we ought to just get rid of it

Reply:
Well then remove the variable altogether
We shouldn't leave something unimplemented